CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/MyCoinView.swift

```diff
@@ -48,6 +49,16 @@ public struct MyCoinView: View {
                 await viewModel.fetchChartPricesData(id: myCoinState.id, timeFrameItem: selectedTimeFrameItem)
             }
         }
+        .onReceive(viewModel.$isSuccess) { isSuccess in
+            if isSuccess {
+                tempSelectedTimeFrameItem = selectedTimeFrameItem
+            } else {
+                selectedTimeFrameItem = tempSelectedTimeFrameItem
+            }
+        }
+        .refreshable {
+            selectedTimeFrameItem = .init(timeFrame: .oneDay)
```

Comment:
Oh so now when we refresh the `Coin Details` screen, we will only refresh the chart region data with 1D filter but we don't trigger to reload the `coinDetail` data anymore? Is this something expected? 🤔

Reply:
My bad 😢 . I added the `fetchCoinDetail` into `refreshable` modifier at https://github.com/nimblehq/nimble-crypto-ios/pull/98/commits/de4774a78c84143af07df2964e1b1ac2e9701ff1.